model/distro/distro.go

```diff
@@ -634,6 +635,7 @@ func (d *Distro) GetResolvedHostAllocatorSettings(s *evergreen.Settings) (HostAl
 		MinimumHosts:           has.MinimumHosts,
 		MaximumHosts:           has.MaximumHosts,
 		AcceptableHostIdleTime: has.AcceptableHostIdleTime,
+		FutureHostPercent:      has.FutureHostPercent,
```

Comment:
It looks like this function isn't used anywhere. What is the intent for this?

Reply:
Fixing rn

Reply:
Fixed. See also https://jira.mongodb.org/browse/EVG-13527